.github/workflows/quality-aarch64.yaml

```diff
@@ -30,11 +31,6 @@ jobs:
           override: true
           components: rustfmt, clippy
 
-      - name: Debug Check (default features)
-        run: |
-          git rev-list origin/main..$GITHUB_SHA | xargs -t -I % sh -c 'git checkout %; cargo check --tests --all --target=${{ matrix.target }}'
```

Comment:
Out of curiosity, what part wasn't working on aarch64?

Reply:
It can't cross build and fails on the build.rs scripts that some scripts have. When we do cross builds we do so using `use-cross` on the github action for Rust toolchain.